aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -73,6 +75,22 @@ class AztecToolbar : FrameLayout, OnMenuItemClickListener {
         }
     }
 
+    override fun onRestoreInstanceState(state: Parcelable?) {
+        val savedState = state as SourceViewEditText.SavedState
```

Comment:
So instead of using internal `SavedState` from `SourceViewEditText` what do you think about moving it to external class, like `AztecSavedState` and using it here, in `SourceViewEditText` and `AztecText`?